Added some spaces to improve readability

Maselko opened the pull request:
It's been a while since last pull request. As finally I have sem. 1 behind, I can finally work on this code. Sorry for little delay.

# Changes made
- 4 spaces indentation rather than 2 spaces indentation as syntax highlighting works better on jupyter notebook readers.
- Spaces between args and some operators to improve readability. I have kept the non-spaced for arguments of functions passed as arguments to other function.
- I have only added spaces and some occasional linebreaks. So all the functions are kept the same(just with more spaces inside).

# Questions / (Typical Students') Suggestions
While adding those spaces I found some stuff unclear to me. I do not mind fixing them but wanted to ask your opinion on them first.
### panel_method.ipynb
- **codecell 4, line 8** : Would you mind describing what '~' thing does as I have searched and found that it is bitwise operator but it's function seems a bit unclear for me.
- **codecell 4, line 13** : Lambda functions are something we have never been taught so either you ask students to figure out or tell them what it does. Some reference or example would be lovely thing to have.

### water_waves.ipynb
- **codecell 3, line 1 and 2** : Is there any particular reason importing them here instead of first cell?
- **codecell 3, line 25 and 26** : This does not work due to previously discussed lack of `ffmpeg` on Windows. I will try making PillowWriter() function working later so it will work on our beloved Windows as well...
- **codecell 4, line 6 and 7** : Same as above.
- **codecell 6, line 2** : '@' some explanation of this '@' usage would be nice as most of us never have seen this before.

Maselko committed:
Added some spaces to improve readability

== Changes made: ==
- 4 spaces indentation rather than 2 spaces indentation as syntax highlighting works better on jupyter notebook readers.
- Spaces between args and some operators to improve readability. I have kept the non-spaced for arguments of functions passed as arguments to other function.
- I have only added spaces and some occasional linebreaks. So all the functions are kept the same(just with more space inside).

weymouth commented on notebooks/water_waves.ipynb:
```diff
@@ -155,29 +142,29 @@
    "outputs": [],
    "source": [
-    "from matplotlib.animation import FuncAnimation\n",
+    "from matplotlib.animation import FuncAnimation, PillowWriter\n",
```
Does this help? Otherwise you shouldn't add it.

weymouth committed:
Updates

I'm happy to go with the 4-space version. I think google colab defaults to 2 spaces with Python, not sure why.

Thanks, for the questions - those are very helpful.
 - `~` is the negation, or bit-flip. So `~mask(x,y)` is checking where `mask(x,y)==False`. I've updated that code to use this more obvious statement.
 - I've added a few links to lambda functions.
The @ operator is actually in the first notebook. I've clarified its use and added a link. I also added a note about the unpacking operator since `quiver(*XY,*UV)` might be odd looking.

 - I was trying to keep all the movie imports with the function itself.
 - The code as written works if I am running in Colab, Windows or Mac.  Can you check?
For running local on Windows, I got a video using the `%matplotlib notebook` backend, but I needed to rewrite the plotting function a little. Let me know how you get on.

weymouth commented:
I'm happy to go with the 4-space version. I think it's google colab that defaults to 2 spaces with Python, not sure why. 

Thanks, for the questions - those are very helpful. 
 - `~` is the negation, or bit-flip. So `~mask(x,y)` is checking where `mask(x,y)==False`. I've updated that code to use this more obvious statement.
 - I've added a few links to lambda functions.
The @ operator is actually in the first notebook. I've clarified its use and added a link. I also added a note about the unpacking operator since `quiver(*XY,*UV)` might be odd looking.

 - I was trying to keep all the movie imports with the function itself. 
 - The code as written works if I am running in Colab, Windows or Mac.  Can you check? For running local on Windows, I got a video using the `%matplotlib notebook` backend, but I needed to rewrite the plotting function a little. Let me know how you get on.

Maselko commented:
Thank you for quick response and clarification on many things.

The video thing works on the Colab on Windows, Mac and Linux, but I guess that's because the functions are executed on server side rather than on your machine. I will have a fiddle with it today and try to come up with some solution for executing them on local machine. Let's see whether this will increase the code size significantly... If there will be some elegant looking alternative solution, I will create another pull request with those.

weymouth commented:
Sounds good, but I'll accept this one as is for now.
